acj/api/ltilaunch.py

```diff
@@ -46,12 +47,18 @@ def lti_auth():
         sess['lti_consumer'] = lti_consumer.id
         lti_user = LTIUser.get_by_tool_provider(lti_consumer, tool_provider)
         sess['lti_user'] = lti_user.id
+        sess['user_oauth_id'] = lti_user.user_oauth_id
```

Comment:
instead of placing all these extra fields in the session, can you instead query the database for the relevant lti table record?